src/tests/stdlib/load_env_file.ab

```diff
@@ -0,0 +1,11 @@
+import { load_env_file, get_env_var, file_write } from "std"
+main {
+    let path = unsafe $mktemp -d /tmp/amber-XXXX$
+    unsafe $cd {path}$
+    unsafe file_write(".env", "TEST=1")
+    load_env_file()
```

Comment:
Once the optional arguments get added can we change the function to accept other path?
```
load_env_file(path = ".env")
```

Reply:
yeah I think that everytime we hadd new feature we will check the various stdlib and improve their code.